src/main/java/net/fabricmc/meta/data/VersionDatabase.java

```diff
@@ -86,7 +87,7 @@ private void loadMcData() throws IOException {
 			return 0;
 		});
 
-		game = minecraftVersions.stream().map(s -> new BaseVersion(s, launcherMeta.isStable(s))).collect(Collectors.toList());
+		game = minecraftVersions.stream().map(s -> new BaseVersion(s, launcherMeta.isStable(s), MinecraftSemverConverter.convert(launcherMeta.majorVersion(s), s))).collect(Collectors.toList());
```

Comment:
should prob make this its own method now, its getting a little big.

Reply:
I can make the change if you'd like